### DISCUSSION TITLE
Release cycle

### DISCUSSION THREAD

**Discussion by HofmannZ** [linkerd/linkerd2#5271] [Q&A] [answered]
We are waiting on path release 2.9.1 with the following fix: #5228.

We've been looking for some indication to when that patch might be released? Any chance there is some sort of release cycle?

Thanks!

**Comment by kleimkuhler** [comment, marked as answer]
@HofmannZ `edge-20.11.4` which released this past Friday has the the main things we wanted to get in for 2.9.1. I expect 2.9.1 will be released next week, or possibly end of this week.
